### PR TITLE
upgrade inflection to 0.4.0

### LINKED ISSUE #301: Update inflection library  to latest version 0.4.0
see #300 for the PR

### PR DESCRIPTION
**Pull request by simkimsia**
see #300 and #301 

closed #301 when merged

### CODE REVIEW

**Comment by simkimsia** [comment]
@suavesav can you merge this?

**Review by suavesav — APPROVED** [review]

**Comment by simkimsia** [comment]
Thank you 😊

**Comment by simkimsia** [comment]
Can someone merge this already reviewed PR? or let me know the feedback if changes are needed? @nnegri @ryochiji  help pls?

**Comment by suavesav** [comment]
Sorry @simkimsia thought you would be able to merge it in once I approved

**Comment by simkimsia** [comment]
I don't think I have the rights to merge

Thank you @suavesav

File: install_requires.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Django>=1.11,<3.0.0
 djangorestframework>=3.8.0,<3.12.0
-inflection==0.3.1
+inflection==0.4.0
```
Comment (ryochiji) [inline install_requires.txt:3]: @simkimsia @suavesav  - As a retroactive point of feedback, requirements in `install_requires` should be as permissive as possible as to reduce conflicting requirements in the upstream app. I don't have context on how strict this requirement should be but should probably be `>=0.4,<0.5` to at least allow for the entire 0.4.x range, or even `>=0.3,<=0.5` unless there are any known issues with those versions. (Also, to be clear, I understand it was already pinned to a specific minor version, so not assigning blame here.)

Comment (simkimsia) [reply install_requires.txt:3]: Copy that do u need me to change? And also I like to expand the range. Cause now there's 0.5.1

**Pull request merged**